cocos/platform/ohos/CCCommon-ohos.cpp

```diff
@@ -0,0 +1,26 @@
+#include "platform/CCPlatformConfig.h"
+#if CC_TARGET_PLATFORM == CC_PLATFORM_OHOS
+
+#include "platform/CCCommon.h"
+#include "platform/ohos/napi/helper/NapiHelper.h"
+#include "CCLogOhos.h"
+#include <stdio.h>
+
+NS_CC_BEGIN
+
+#define MAX_LEN         (cocos2d::kMaxLogLen + 1)
+
+void MessageBox(const char * pszMsg, const char * pszTitle) {
+    std::string msg(pszMsg);
+    std::string title(pszTitle);
+    JSFunction::getFunction("DiaLog.showDialog").invoke<void>(msg, title);
```

Comment:
Should `DiaLog.showDialog` be `Dialog.show` ?  `Dialog` is a word and doesn't need to make the `L` uppercase.
And `show` has already been in the scope of Dialog, the name of `showDialog`  is a little redundant.

Reply:
the DiaLog.showDialog  is defined at NapiHelper.ts in template project, just a string used to point to the function. We have declared many functions with similar styles in NapiHelper.ts. So I think there is no need to change this.